core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -506,6 +506,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
     val (sched, ts) = SparkContext.createTaskScheduler(this, master)
     _schedulerBackend = sched
     _taskScheduler = ts
+    logDebug("Starting DAGScheduler")
```

Comment:
Hi Jacek:

My only concern with putting log in DAGScheduler's constructor is that the logger might not have been initialized when constructor is called.